config/api.yml

```diff
@@ -2667,20 +2667,13 @@
         :identifier: region
       - :name: edit
         :identifier: region_edit
-      - :name: delete
-        :identifier: region_delete
     :resource_actions:
       :get:
       - :name: read
         :identifier: region
       :post:
       - :name: edit
         :identifier: region_edit
```

Comment:
Just as an aside, we can't edit the region's `region_id` attribute, correct?

Reply:
It can't edit the `'id` field, but currently the `region` field is editable. I don't see a `region_id` field.

Reply:
Yeah, that's the one. Really the only thing that should be editable in this table is the `description`

Reply:
@djberg96 can you add the other attributes to `INVALID_REGIONS_ATTRS` as a part of this change?

Reply:
@carbonin ok, updated.